CONFIG.md

```diff
@@ -374,12 +373,11 @@ This is an example of the settings for the validate tasks:
   }
   "phplint": {
     "dir": [
-      "<%= config.srcPaths.drupal %>/**/*.php",
```

Comment:
I was initially unsure of removing themes, but in practice I have found a default assumption that rigid coding standards can be applied to templates is fairly difficult. In that light, better to have a project opt-in.